engine/src/main/java/org/hibernate/validator/internal/constraintvalidators/bv/time/AbstractEpochBasedTimeValidator.java

```diff
@@ -0,0 +1,67 @@
+/*
+ * Hibernate Validator, declare and validate application constraints
+ *
+ * License: Apache License, Version 2.0
+ * See the license.txt file in the root directory or <http://www.apache.org/licenses/LICENSE-2.0>.
+ */
+package org.hibernate.validator.internal.constraintvalidators.bv.time;
+
+import java.lang.annotation.Annotation;
+import java.time.Clock;
+
+import javax.validation.ClockProvider;
+import javax.validation.ConstraintValidator;
+import javax.validation.ConstraintValidatorContext;
+
+import org.hibernate.validator.internal.util.logging.Log;
+import org.hibernate.validator.internal.util.logging.LoggerFactory;
+
+/**
+ * Base class for all time validators that use an epoch to be compared to the time reference.
+ *
+ * @author Alaa Nassef
+ * @author Guillaume Smet
+ */
+public abstract class AbstractEpochBasedTimeValidator<C extends Annotation, T> implements ConstraintValidator<C, T> {
+
+	private static final Log log = LoggerFactory.make();
```

Comment:
I know it's done inconsistently, but I'd prefer to have this named `LOG` (that's how I do it in new code) as it's a constant.